docs/cli.md

```diff
@@ -2,6 +2,7 @@
 
 BigFlow package offers a command-line tool called `bigflow`.
 It lets you run, build, and deploy your workflows from command-line on any machine with Python.
+It is also possible to generate GCP Logging link.
```

Comment:
Remove this sentence and add "debug" to the sentence above (for example).

Reply:
I wouldnt mix logging with debugging.

Reply:
That sentence is on the very top of the document, next to the **very** brief description of the CLI. It looks out of context:

![Zrzut ekranu 2020-10-15 o 15 11 54](https://user-images.githubusercontent.com/9019407/96131566-a43c6880-0ef9-11eb-88fc-10cf8af83dc3.png)

Reply:
@anjensan @bartoszwalacik  what do you think?

Reply:
I agree with you that this sentence could be better. But mixing logging with debugging is not ok.